application/client/src/app/ui/views/toolbar/search/input/component.ts

```diff
@@ -51,6 +53,16 @@ export class ViewSearchInput
 
     public ngAfterContentInit(): void {
         this.progress = new Progress(this.session);
+        const searching: boolean | undefined = this.session.search.state().searching;
+        if (searching !== undefined) {
+            if (searching) {
+                this.progress.start();
+            } else {
+                this.progress.hidden = false;
+                this.progress.setFound(this.session.search.map.len());
+                this.detectChanges();
```

Comment:
view isn't created yet (you are in `AfterContentInit`), you should not call detection of changes

Reply:
You're right. Didn't consider that. Should've moved it into `ngAfterViewInit` instead